api/src/main/java/com/messagebird/objects/ErrorReport.java

```diff
@@ -1,32 +1,40 @@
 package com.messagebird.objects;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 /**
  * When MessageBird returns a 4xx, you will find a list of any error codes in your return dataset.
  * you will receive a list of errors from the API in such case.
  *
  * Created by rvt on 1/5/15.
  */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class ErrorReport {
     private Integer code;
     private String description;
     private String parameter;
+    private String message;
 
     public ErrorReport() {
     }
 
-    public ErrorReport(Integer code, String description, String parameter) {
+    public ErrorReport(Integer code, String description, String parameter, String message) {
         this.code = code;
         this.description = description;
         this.parameter = parameter;
+        this.message = message;
     }
 
     @Override
     public String toString() {
-        return "ErrorReport{" +
-                "code=" + code +
-                ", description='" + description + '\'' +
-                ", parameter='" + parameter + '\'' +
-                '}';
+        String str =  "ErrorReport{code=" + code;
+        if (message != null) {
```

Comment:
Beside null check, I think you should check with isEmpty method as well.

Reply:
fixed